FreeType/Lens.hs

```diff
@@ -0,0 +1,253 @@
+{-# LANGUAGE FlexibleInstances #-}
```

Comment:
This is a rather tough question, because on one hand the library has **a lot** of overlapping field names, so you'd pretty much want to use `[micro]lens`, however if you actually do, you cannot `peek` with them, allowing for some hilarious one-liners like
```
\face -> do
   btmp <- view bitmap <<$>> peek . view glyph =<< peek face
```

Though now that I think of it I should just regex all the `template-haskell` `Name`s in modules, you're right.